src/autogen_module/numpy_eigen_export_module.cpp

```diff
@@ -253,6 +253,8 @@ BOOST_PYTHON_MODULE(libnumpy_eigen)
 {
 	using namespace boost::python;
 	// Without this import, the converter will segfault
+#undef NUMPY_IMPORT_ARRAY_RETVAL
+#define NUMPY_IMPORT_ARRAY_RETVAL
```

Comment:
No need to change but more idiomatic would be to use #ifndef ... (just like the input guards).